packages/devextreme/js/events/double_click.js

```diff
@@ -57,9 +57,11 @@ const DblClick = Class.inherit({
 
     remove: function() {
         this._handlerCount--;
+
         if(this._handlerCount <= 0) {
             this._forgetLastClick();
             eventsEngine.off(domAdapter.getDocument(), NAMESPACED_CLICK_EVENT);
+            this._handlerCount = 0;
```

Comment:
I'm not going to review this change but I believe there should be a test for this